vignettes/excusing-assignments.Rmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "Excusing Assignments"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Excusing Assignments}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(gradebook)
+```
+
+Gradebook supports instructors "excusing" certain students from certain assignments. 
+
+## What does it mean to "excuse" an assignment?
+
+When an instructor "excuses" an assignment, this is equivalent to allowing the student to drop the assignment. This means that the assignment will not be counted towards the student's final grade. This is not equivalent to giving the student a 0 on an assignment. Excusing a student from an assignment removes both the student's score and the assignment's points possible from consideration for that student.
+
+### An Example
+Mathematically, consider the common scenario where the student's grade equals the number of points student received on the assignments divided by the number of points possible on the assignments. 
+
+For example, suppose that we have a category named Homework with assignments HW 1, HW 2, and HW 3. Suppose that each assignment is worth 10 points and a student Jenny received 2, 6, and 8 points on the homework respectively. Then Jenny would receive homework score (2 + 6 + 8) / (10 + 10 + 10) = 8/15.
+
+In our example, suppose that we excused another student Timmy from Homework 1. If Timmy got 5 and 7 points on homework 2 and 3, then Timmy's Homework score would be (5 + 7)/(10 + 10) = 0.6. 
+
+
+## How to Excuse Assignments
+
+Our notation for excusing assignments is molded around Canvas's notation for excusing assignments. Thus, Canvas users can simply excuse an assignment in Canvas and Gradebook will recognize the assignment as excused. 
+
+Users of other grade aggregators who wish to excuse assignments should set the assignment's score value to "EX" for it to be excused in Gradebook. 
+
+## Notes on the Interaction between Dropped Assignments and Category Drops
```

Comment:
This should read: "Notes on the Interaction between Excused Assignments and Category Drops